src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -0,0 +1,71 @@
+import { Knapp } from 'nav-frontend-knapper';
+import { Systemtittel } from 'nav-frontend-typografi';
+import * as React from 'react';
+
+import { hentVedtaksbrev, IVedtaksBrev } from '../../../api/oppsummeringvedtak';
+import { IFagsak } from '../../../typer/fagsak';
+import { Ressurs, RessursStatus } from '../../../typer/ressurs';
+
+interface IVedtakProps {
+    fagsak: IFagsak;
+}
+
+interface IOppsummeringFrameProps {
+    content: string;
+}
+
+const OppsummeringFrame: React.FunctionComponent<IOppsummeringFrameProps> = ({ ...props }) => {
+    const iframe = React.useRef();
+
+    React.useEffect(() => {
+        iframe.current.contentDocument.body.innerHTML = props.content;
+    });
+
+    return <iframe className="iframe" ref={iframe} />;
+};
+
+const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) => {
+    const [brev, setBrev] = React.useState('Genererer forhåndsvisning...');
+
+    React.useEffect(() => {
+        hentVedtaksbrev(fagsak.id)
+            .then((response: Ressurs<IVedtaksBrev>) => {
+                if (response.status === RessursStatus.SUKSESS) {
+                    setBrev(response.data.content);
+                } else {
+                    setBrev(
+                        'kunne ikke generere forhåndsvisning'.concat(
```

Comment:
We should depend on the error message from ressurs, hence you can just do something like `setBrev(response.melding)`.

Reply:
done